### DISCUSSION TITLE
Feature request: D&D 3.5e Statblock

### DISCUSSION THREAD

**Discussion by shadychild** [Obsidian-TTRPG-Community/fantasy-statblocks#186] [Feature Requests]
Hi,
I have D&D 3.5 SRD (attached) and it would be great to build a 3.5e Statblock from it.
[monster.txt](https://github.com/valentine195/obsidian-5e-statblocks/files/7882395/monster.txt)

**Comment by sigrunixia** [comment]
Hi @shadychild 
There is this 3.5e Statblock made by @Trellinane

https://github.com/valentine195/fantasy-statblocks/discussions/166

As for the 3.5SRD, we'd need it to be in .json format, and matched up to a statblock layout appropriately. 
There are some good converters out there that can get you pretty close from XML to .json, but there will be some cleanup work to do. Our plate is full, but I will go ahead and throw help wanted on this in case anyone else wants to pick this up.

**Comment by sigrunixia** [comment]
Converting this to a Discussion.

**Reply by sigrunixia** [reply]
Conversion Complete
